experimental/generated/beats/fields.ecs.yml

```diff
@@ -1573,6 +1573,60 @@
         Note that when the file name has multiple extensions (example.tar.gz), only
         the last one should be captured ("gz", not "tar.gz").'
       example: png
+    - name: file.elf.creation_date
+      level: extended
+      type: date
+      description: Extracted when possible from the file's metadata. Indicates when
+        it was built or compiled. It can also be faked by malware creators.
+      default_field: false
+    - name: file.elf.exports
+      level: extended
+      type: object
+      description: List of exported element names and types
+      default_field: false
+    - name: file.elf.header
```

Comment:
I might consider collapsing things from `header` into the top-level attributes of `elf` itself and exposing them as separate fields. We could ideally have some stuff like:

- `elf.byte_order` - (big endian v. little endian)
- `elf.architecture` - (arm, amd64, x86, etc)
- `elf.cpu_type` - (Intel, PowerPC, RISC, etc.)